apps/logging/signals.py

```diff
@@ -40,9 +37,7 @@
 @receiver(app_authorized)
 def handle_token_created(sender, request, token, **kwargs):
     # Get auth flow dict from session for logging
-    auth_flow_dict = get_session_auth_flow_trace(request)
-
-    token_logger.info(get_event(Token(token, action="authorized", auth_flow_dict=auth_flow_dict)))
+    token_logger.info(Token(token, action="authorized", auth_flow_dict=get_session_auth_flow_trace(request)))
```

Comment:
So this is a good example of a place where we do have access to the `request` object but we aren't using the `RequestLogger` at all. We should actually just be using the one attached to the `request` object:
`request._logger.info(...`

Reply:
> @JFU-GIT This is looking great so far! Going to do a side by side logging comparison in local dev next.
> 
> I'm thinking that the following changes can be done as part of the scope for this PR?
> 
> In the @nbragdon original/related PR, he converted a few logging functions over to be inline instead, as an example of using the new logging class. These were the `log_get_and_update_user()` and `log_create_beneficiary_record()` functions that were located in apps/mymedicare_cb/loggers.py and removed.
> 
> For example, the following is a link to the start of the code related to replacing `log_get_and_update_user()`:
> 
> https://github.com/CMSgov/bluebutton-web-server/blob/2f321fbf2962637893395131e1943d6bd90c83f7/apps/mymedicare_cb/models.py#L62
> 
> There are other similar logging functions that could or should be replaced. Performed a search for "import log_" in the code base and found the following files. The ./apps/logging/management/commands/log_global_state_metrics.py should not be updated, though.
> 
> Files list:
> 
> ./apps/accounts/views/oauth2_profile.py
> ./apps/mymedicare_cb/authorization.py
> ./apps/mymedicare_cb/views.py
> ./apps/logging/management/commands/log_global_state_metrics.py # Doesn't need updating...
> ./apps/fhir/server/authentication.py
> ./apps/fhir/bluebutton/views/generic.py

thanks for locate those calling points where calls could be refactored to the new log method signature which is cleaner and uniformed...., was trying to avoid extensive code changes - but that seems to be part of the value.